src/data/helpers.test.tsx

```diff
@@ -0,0 +1,37 @@
+import test from 'tape'
+import { importSingleFile, importFiles, expandCollection } from './helpers'
+
+import testItem1 from './testCollection/item1.json'
+import testItem2 from './testCollection/item2.json'
+
+test('importSingleFile', async t => {
+  const data = await importSingleFile('./testCollection/item1.json')
+
+  t.deepEqual(data, testItem1)
+  t.end()
+})
+
+test('importFiles', async t => {
+  const data = await importFiles('./testCollection/', ['item1', 'item2'])
+
+  t.deepEqual(data, [testItem1, testItem2])
+  t.end()
+})
+
+test('expandCollection', async t => {
+  const data = {
+    someKey: 'someValue',
+    testCollection: ['item1', 'item2'],
+  }
+
+  const dataExpanded = await expandCollection(
+    'testCollection',
+    './testCollection/',
+    data,
+  )
+  t.deepEqual(dataExpanded, {
+    someKey: 'someValue',
+    testCollection: [testItem1, testItem2],
+  })
+  t.end()
+})
```

Comment:
If I understand this and the definition correctly, expandCollection expects an object with a key equal to its first argument, name, which has a list of strings as values. The strings are interpreted as unique IDs of entries in a folder collection (actually as filenames). The second argument specifies the collection (actually the folder). The function will replace each string id with the JSON object.

Thoughts:
- The field holding references may not be at the top level of the object. Maybe support passing in a path (like R.path) instead of a prop name?
- There may be multiple references in an object that need to be replaced. Currently you need to call the function multiple times to do this. I think this is ok, actually... the caller (currently getTeam in team/index.tsx) can specify which fields are  references and need to be replaced

Reply:
I changed the implementation a bit. It does not rely on the filenames any more and only uses the uuid to match collection items. Further, the second argument is a list of already imported items.

Concerning your thoughts:
* Great idea! I changed the function to use `R.path`.
* I agree. Sounds like a plan.